client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1298,24 +1315,73 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
       shuffleId: Int,
       slotsToDestroy: WorkerResource): Unit = {
     val shuffleKey = Utils.makeShuffleKey(appUniqueId, shuffleId)
-    val parallelism = Math.min(Math.max(1, slotsToDestroy.size()), conf.clientRpcMaxParallelism)
```

Comment:
after this change, the conf `celeborn.client.rpc.maxParallelism` will no longer be used anywhere. it safe to remove?

Reply:
removed